deeppavlov/core/commands/utils.py

```diff
@@ -9,19 +9,17 @@ def set_usr_dir(config_path: str, usr_dir_name='USR_DIR') -> PurePath:
     Make a serialization user dir.
     """
     config = read_json(config_path)
-    try:
```

Comment:
try...except KeyError is more pythonic way to deal with dictionary keys checking than if...else